src/helpers.js

```diff
@@ -62,3 +64,60 @@ export const openEditor = async () => {
     checkIsOpen()
   })
 }
+
+export const checkResourceUrlExists = async (url) => {
+  try {
+    const result = await fetch(url, {method: 'HEAD'})
+    return result.status === 200
+  } catch {
+    return false
+  }
+}
+
+export const checkExternalUrlExists = async (url) => {
+  try {
+    const command = `wget -q --method=HEAD ${url}`
+    const res = await window.codioIDE.remoteCommand.run(command)
+    return res.code === 0
```

Comment:
I am not sure if `wget` is installed in all boxes
The common command not found code is `127` - maybe we could add extra logic here

Reply:
added logs to console if fetch or remote command failed or not succeeded